articles/machines/toffoli-simulator.md

```diff
@@ -0,0 +1,52 @@
+---
+# Mandatory fields. See more on aka.ms/skyeye/meta.
+title: Quantum Development Kit Toffoli Simulator | Microsoft Docs 
+description: Overview of Microsoft's Quantum Development Kit Toffoli Simulator 
+author: alan-geller
+ms.author: ageller@microsoft.com 
+ms.date: 1/16/20179
+ms.topic: article
+uid: microsoft.quantum.machines.toffoli-simulator
+---
+
+# Quantum Development Kit Toffoli Simulator
+
+The Quantum Development Kit provides a Toffoli simulator,
+which is a special-purpose simulator that can simulate quantum algorithms
+that are limited to X, CNOT, and multi-controlled X quantum operations
+(all classical logic and computations are available).
+This simulator can be used to execute and debug a limited set of
+quantum algorithms written in Q# on your computer.
```

Comment:
I would give an example of what kind of algorithms this is useful for - less of a cliffhanger but more useful :-)

Reply:
Done.